MapboxNavigation/NavigationViewController.swift

```diff
@@ -561,6 +561,14 @@ extension NavigationViewController: RouteControllerDelegate {
         }
         return advancesToNextLeg
     }
+    
+    @objc public func routeController(_ routeController: RouteController, didEnterTunnelAt location: CLLocation?) {
+        if let _ = location {
```

Comment:
The fact we're not using `location` here might clue us in that it may not be necessary to send this param over.

Reply:
Sure!